bin/createSwooleTimerTable.php

```diff
@@ -10,6 +10,7 @@
 
     $timerTable->column('worker_pid', Table::TYPE_INT);
     $timerTable->column('time', Table::TYPE_INT);
+    $timerTable->column('fd', Table::TYPE_INT);
```

Comment:
Isn't this a breaking change? What will happen for existing Octane installations that don't have this column?

Reply:
It should not be BC, it will be overwritten after updating the version.

Reply:
You mean it should be BC? Because if it isn't then we need to target this PR to master.

Reply:
It's not. 
It is an enhancement.

Reply:
I feel like there's a slight language barrier here. I'm trying to determine if apps using Octane NOW will break if we ship this. Can you confirm they won't break saying "they won't break"?

Reply:
If a new version is released after the merger, the bin directory will be updated with the version. Will this cause any problems?

Reply:
That's what I'm asking you 😅 

You're setting a value in this table here: https://github.com/laravel/octane/pull/510/files#diff-d72d084dbc08724cd72e4665b6da8495735c3f21b16e2ea8c6ac669fb56f3b0cR110

But existing Octane apps on the 1.x release won't have this column. I have no idea what Swoole will do if that happens. That's why I want to make sure that nothing breaks for people who don't have this column.

Reply:
From my understanding, Swoole only offers the generic table framework and the discussed change is Octane specific. And since Swoole tables are in-memory only, this should be BC as long as Octane is restarted after an update.

Reply:
@Namoshek thanks for that. Since we recommend that in the docs, I feel that this change is okay 👍 

https://laravel.com/docs/9.x/octane#reloading-the-workers